### PR TITLE
Update file handling article.

### PR DESCRIPTION
**Pull request by jpmedley**
Feature likely shipping in Chrome 95.

### CODE REVIEW

**Comment by jpmedley** [comment]
@tomayac I may have jumped the gun a bit. It's not clear that this is shipping in 95.

File: src/site/content/en/blog/file-handling/index.md [hunk 1]
```diff
@@ -43,30 +42,14 @@ Examples of sites that may use this API include:
 | ---------------------------------------- | ------------------------ |
 | 1. Create explainer                      | [Complete][explainer]    |
 | 2. Create initial draft of specification | Not started              |
-| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
-| 4. **Origin trial**                      | **[In progress][ot]**    |
-| 5. Launch                                | Not started              |
+| 3. Gather feedback & iterate on design   | Complete                 |
+| 4. Origin trial                          | Complete                 |
+| 5. **Launch**                            | Chrome 95                |
```
Comment (tomayac) [inline src/site/content/en/blog/file-handling/index.md:47]:
```suggestion
| 5. **Launch**                            | **Chromium&nbsp;95**           |
```

**Review by tomayac — APPROVED** [review]
LGTM, approval

File: src/site/content/en/blog/file-handling/index.md [hunk 2]
```diff
@@ -7,15 +7,14 @@ Description: |
   Register an app as a file handler with the operating system
   and open files with their proper app.
 date: 2020-10-22
-updated: 2021-07-29
+updated: 2021-09-22
```
Comment (tomayac) [inline src/site/content/en/blog/file-handling/index.md:10]:
```suggestion
updated: 2021-12-21
```

File: src/site/content/en/blog/file-handling/index.md [hunk 3]
```diff
@@ -42,19 +43,14 @@ Examples of sites that may use this API include:
 | ---------------------------------------- | ------------------------ |
 | 1. Create explainer                      | [Complete][explainer]    |
 | 2. Create initial draft of specification | Not started              |
-| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| 3. Gather feedback & iterate on design   | Complete                 |
 | 4. Origin trial                          | Complete                 |
-| 5. Launch                                | Not started              |
+| 5. **Launch**                            | **Chromium&nbsp;95**     |
```
Comment (tomayac) [inline src/site/content/en/blog/file-handling/index.md:48]:
```suggestion
| 5. **Launch**                            | **Chromium&nbsp;98**     |
```

**Pull request merged**